eng/clipack/Common.projitems

```diff
@@ -85,4 +85,22 @@
            vs regex: $(FileSignatureRegex)" />
   </Target>
 
+  <Target Name="_CheckMaxFileSize" Condition="'$(ContinuousIntegrationBuild)' == 'true'">
```

Comment:
```suggestion
  <Target Name="_CheckMaxFileSize">
```

No reason to check it only in CI. Can get the error locally too.

Reply:
I guess one reason to not do this locally is because we are calling into `pwsh`, which might not be available on a dev's Unix machine.